src/main/java/project/service/AbstractService.java

```diff
@@ -37,6 +37,7 @@ protected Connection getConnectionToDatabase() throws URISyntaxException, SQLExc
         // TODO: viljum við nota '!connection.isValid(VALID_TIMEOUT)'
         if  (connection == null || connection.isClosed() || !connection.isValid(VALID_TIMEOUT))
         {
+            // TODO: use system env. variables to store database info, and when that is done, change the database user and passw @ Heroku
```

Comment:
can you do this tasks when you can:
- https://app.asana.com/0/52964612936482/57722993923201
- https://app.asana.com/0/52964612936482/57722993923199
?